app/controllers/application_controller.rb

```diff
@@ -8,7 +8,7 @@ class ApplicationController < ActionController::Base
   protected
 
   def configure_devise_permitted_parameters
-    registration_params = [:name, :address ,:phone, :description, :email, :image, :password, :password_confirmation]
+    registration_params = [:name, :patronymic, :surname, :address ,:phone, :description, :email, :image, :password, :password_confirmation]
```

Comment:
Line is too long. [139/80]<br>Space missing after comma.<br>Space found before comma.